config/api.yml

```diff
@@ -1064,6 +1064,42 @@
       :delete:
       - :name: delete
         :identifier: orchestration_template_remove
+  :physical_servers:
+    :description: Physical Servers
+    :identifier: physical_server
+    :options:
+    - :collection
+    :verbs: *gp
+    :klass: PhysicalServer
+    :subcollections:
+    :collection_actions:
```

Comment:
missing :get: section in collection and resource section with appropriate identifiers for listing (show_list) and fetching (show) a physical server.

also, should implement the query post action for bulk queries.